src/components/Posts/PostItem.tsx

```diff
@@ -141,6 +149,9 @@ const PostItem = ({ post }: PostProps) => {
             {post.numberOfLikes}
           </Button>
           <Button
+            onClick={() => {
+              replyModalPopup();
+            }}
```

Comment:
Can we get rid of the function definition and inline this instead? If the logic isn't too complicated I think it's fine to inline it

```suggestion
            onClick={() => {
              setOpenModal(true);
            }}
```

Reply:
Yeah will make this change, I think I was trying something with multiple functions but removed that part so this should have been kept in-line, thanks